dev/release/release-config.jsonc

```
@@ -10,15 +10,15 @@
  */
 {
     // Captain information
-    "captainSlackUsername": "Joe Chen",
+    "captainSlackUsername": "joe",
     "captainGitHubUsername": "unknwon",
     // Release versions
-    "previousRelease": "3.40.0",
-    "upcomingRelease": "3.40.1",
+    "previousRelease": "3.40.1",
+    "upcomingRelease": "3.41.0",
     // Release dates
-    "oneWorkingWeekBeforeRelease": "25 May 2022 10:00 PST",
-    "oneWorkingDayBeforeRelease": "31 May 2022 10:00 PST",
-    "releaseDate": "01 June 2022 10:00 PST",
+    "oneWorkingWeekBeforeRelease": "15 May 2022 10:00 PST",
```

```suggestion
    "oneWorkingWeekBeforeRelease": "15 June 2022 10:00 PST",
```

Shouldn't this be June?

---

Start to wonder why can't these be calculated automatically 😂